edk2toollib/uefi/edk2/parsers/base_parser.py

```diff
@@ -274,11 +274,11 @@ def ProcessConditional(self, text):
             return True
 
         elif(tokens[0].lower() == "!ifdef"):
-            self.PushConditional((tokens[1].count("$") == 0))
+            self.PushConditional(not (self.LocalVars.get(tokens[1]) is None))
```

Comment:
I think is not None is a little more pythonic

Reply:
Though I'm open to keeping it as is if you have strong feelings about it

Reply:
I agree with Matt.

Reply:
I agree.  I will change to "is not None"